apps/src/applab/RedirectDialog.jsx

```diff
@@ -69,12 +67,24 @@ export default class RedirectDialog extends React.Component {
     }
 
     return (
-      <Dialog title={title} isOpen={this.state.isOpen}>
-        <Body>
+      <Dialog title={title} isOpen handleClose={this.props.handleClose}>
+        <Body className={'ui-test-redirect-notice'}>
           {body}
           {footer}
         </Body>
       </Dialog>
     );
   }
 }
+
+export const UnconnectedRedirectDialog = RedirectDialog;
```

Comment:
Out of curiosity, why do we rename this to UnconnectedRedirectDialog during export only to rename it back to RedirectDialog later?

Reply:
This allows us to also use just the UnconnectedRedirectDialog when we want to test things without connecting to redux state - in this case in the RedirectDialog.story.jsx. We want the story to live outside of state since we are just using it in a stateless form in the storybook, so having the unconnected component exported allows us to do that while also have the default connected component.

Reply:
I think we can pull in connected stories into storybook (using `.withReduxStore()`). No preference on which option is better!